torch/csrc/Exceptions.cpp

```diff
@@ -32,6 +32,38 @@ std::string processErrorMsg(std::string str) {
 
   // Translate Aten types to their respective pytorch ones
   std::vector<std::pair<std::string, std::string>> changes {
+    {"Variable[SparseCUDAByteType]", "torch.cuda.sparse.ByteTensor"},
```

Comment:
This is getting really out of hand. Suggestions on how to make this more maintainable are welcome! 

I didn't really want to generate all of this as a part of the codegen.

Reply:
We don't need the tensor strings anymore, right?

Reply:
I think we still might need them because some ATen functions operate on unpacked tensors